HousingSearchApi/V1/Domain/ES/QueryablePerson.cs

```diff
@@ -1,34 +1,8 @@
 using System.Collections.Generic;
 using Nest;
 
-namespace HousingSearchApi.V1.Infrastructure
+namespace HousingSearchApi.V1.Domain.ES
```

Comment:
I think it might be better to rename **ES -> ElasticSearch** ? Also, are these Domain entities? If not, then maybe they should live outside of the Domain namespace to avoid confusion?

Reply:
I can rename ES to ElasticSearch.
As for whether or not they are domain entities...I am open to suggestions - where would you like me to move them?

Reply:
Perhaps to the Gateway folder?